tests/rptest/services/kerberos.py

```diff
@@ -349,13 +369,61 @@ def metadata(self, principal: str):
         kinit_args = f"-kt {self.keytab_file} -c {client_cache} {principal}"
         kinit_cmd = f"kinit -R {kinit_args} || kinit {kinit_args}"
         sasl_conf = f"-X security.protocol=sasl_plaintext -X sasl.mechanisms=GSSAPI '-Xsasl.kerberos.kinit.cmd={kinit_cmd}' -X sasl.kerberos.service.name=redpanda"
-        res = self.nodes[0].account.ssh_output(
-            cmd=
-            f"KRB5_TRACE=/dev/stderr KRB5CCNAME={client_cache} kcat -L -J -b {self.redpanda.brokers()} {sasl_conf}",
-            allow_fail=False,
-            combine_stderr=False)
-        self.logger.debug(f"Metadata request: {res}")
-        return json.loads(res)
+        try:
+            res = self.nodes[0].account.ssh_output(
+                cmd=
+                f"KRB5_TRACE=/dev/stderr KRB5CCNAME={client_cache} kcat -L -J -b {self.redpanda.brokers()} {sasl_conf}",
```

Comment:
The `KRB5_TRACE` is nice for debugging but can definitely clutter up the logs.  How do you feel about removing this?

Reply:
It's piped to `stderr`, which isn't combined. I think you only get that trace if there's a failure on the command?

Perhaps it's in the logs elsewhere?

Reply:
I think that's more what I was meaning.  If a test errors out, this tends to clutter up the logs quite a bit.  But it's probably helpful to keep around so I'm fine if it's kept.

Reply:
Debug logs have this tendency to be both too much and too little. It's often easier to filter than it is to reproduce.